modules/installation-configure-proxy.adoc

```diff
@@ -34,6 +32,13 @@ endif::bare-metal[]
 .Prerequisites
 
 * An existing `install-config.yaml` file.
+// TODO: xref (../../installing/install_config/configuring-firewall.adoc#configuring-firewall)
+* Review the sites that your cluster requires access to and determine whether any need to bypass the proxy. By default, all cluster egress traffic is proxied, including calls to hosting cloud provider APIs. Add sites to the Proxy object's `spec.noProxy` field to bypass the proxy if necessary.
++
+[NOTE]
+====
+The Proxy object's `status.noProxy` field is populated by default with the instance metadata endpoint (`169.254.169.254`) and with the values of the `networking.machineCIDR`, `networking.clusterNetwork.cidr`, and `networking.serviceNetwork` fields from your installation configuration.
```

Comment:
I think we're using _field_ for UI elements and _parameter_ for CRs.

Reply:
@jboxman Do we have guidance on this anywhere? I'm poking around now in the repo and see a mixed bag. I'll probably leave it as is for now to be consistent since I use field throughout the rest of the proxy docs, but I don't mind coming back to update later to be consistent if we determine that parameter is the appropriate term.

Reply:
@openshift/team-documentation ^^

Reply:
@bergerhoffer, Jason's right. Here's what the ISG says about the two:

> Field
 A rectangular box in which users can type text.

> Values of the following elements in
text:
• API structure parameters
• Command parameters and
arguments
• Configuration parameters
• Environment variables
• Keywords
• Registry variables
...
**Parameter values**: yes|no|maybe

Reply:
@kalexand-rh Hmm I actually don't see those entries in my IBM style guide. But I guess what I was really curious about is what YAML actually refers to these as, as sometimes the terminology is dependent on the language.

Reply:
The first is on page 209, and the second is on page 106.

Reply:
I don't have a hard copy, but a copy at Safari online. But I searched for these words and didn't find them.

Reply:
I have a PDF. What's your copyright date? Mine's 2012.

Reply:
My copyright date is the same. Would you mind emailing me the PDF just so I can see it in there?

Reply:
It's licensed, so I can't send you the whole thing. :(